packages/validation/docs/valibot-api-index.md

```
@@ -0,0 +1,1722 @@
+const NullableStringSchema = v.nullable(
+  v.pipe(
+    v.string(),
+    v.transform((input) => input.length)
+  )
+);
+
+// Infer non nullable string input type
+type NonNullableStringInput = v.InferNonNullableInput<
+  typeof NullableStringSchema
+>; // string
+  ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)